File: herbstluftwm/.config/herbstluftwm/python/startup.py
import os
from color import color


def set_background():
    os.system("xsetroot -solid '" + color['blue'] + "'")


def start_panel():
    print('starting panel...')
    os.system('$XDG_CONFIG_HOME/polybar/launch.sh mainbar-herbst &')


def start_applications():
    command = 'silent new_attr bool my_not_first_autostart'
    exitcode = os.system('herbstclient ' + command)

    if exitcode == 0:
        os.system('picom &')
        os.system('feh --no-fehbg --bg-scale "$HOME/Pictures/dark-leaves.jpg" &')
        os.system('sxhkd -c $XDG_CONFIG_HOME/sxhkd/common_sxhkdrc &')

File: herbstluftwm/.config/herbstluftwm/php/Startup.php
<?php

class Startup
{
    private Color $color;

    public function __construct(Color $color)
    {
        $this->color = $color;
    }

    public function setBackground()
    {
        exec("xsetroot -solid {$this->color->getColor('blue')}");

    }

    public function startPanel()
    {
        exec(getenv('XDG_CONFIG_HOME') . '/polybar/launch.sh mainbar-herbst > /dev/null &');
    }

    public function startApplications()
    {
        exec('picom > /dev/null &');
        exec('feh --no-fehbg --bg-center ' . getenv('HOME') . '/Pictures/dark-leaves.jpg > /dev/null &');
        exec('sxhkd -c ' . getenv('XDG_CONFIG_HOME') . '/sxhkd/common_sxhkdrc > /dev/null &');
    }
}